src/tokenizer/main.py

```diff
@@ -251,6 +275,10 @@ def val(t: Tok, quote_word: bool = False) -> Any:
             v = val(t)
             if v is not None:
                 d["v"] = v
+            if t.original is not None:
+                d["o"] = t.original
+            if t.origin_spans is not None:
+                d["s"] = cast(str, t.origin_spans)
```

Comment:
This cast could be avoided if d is declared above as `d: Dict[str, Union[str, List[int]]`

Reply:
Thanks! Fixed